OneCommunityGlobal/HighestGoodNetworkApp
# Yongjian remove admin or owner ability to delete themselves

# Description
<img width="742" alt="Screenshot 2023-09-21 at 1 00 59 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/68c48932-3d68-449f-8d74-01b857d9e9d7">

## Main changes explained:
- Made admin cannot delete themselves and owner cannot delete themselves nor the admin through the user management functionality.

## How to test:
1. check into current branch
2. do `npm run start:local` to run this PR locally
3. log as admin user
4. go to Dashboard -> Other Links -> User Management -> search for your admin role name -> check to ensure delete button is not visible
5. go to Dashboard -> Other Links -> User Management -> search for your owner role name -> check to ensure delete button is not visible for owner nor admin role

## Screenshots or videos of changes:
For Admin role:

<img width="1710" alt="Screenshot 2023-09-21 at 10 55 22 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/4711fb83-ceb7-4e05-9c65-0f859c8c91fa">

For Owner role:

<img width="1710" alt="Screenshot 2023-09-21 at 10 55 58 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/8001c31b-b747-43e4-85ed-7cbee4140ab5">
